modules/service/src/main/resources/db/migration/V0908__itc_asterism.sql

```diff
@@ -0,0 +1,21 @@
+-- Let's drop the itc values, they will be recalculated soon
+TRUNCATE TABLE t_itc_result;
+
+-- Drop individual columns
+ALTER TABLE t_itc_result
+  DROP CONSTRAINT t_itc_result_pkey,
+  DROP COLUMN c_target_id,
+  DROP COLUMN c_sci_exposure_time,
+  DROP COLUMN c_sci_exposure_count,
+  DROP COLUMN c_sci_signal_to_noise,
+  DROP COLUMN c_acq_exposure_time,
+  DROP COLUMN c_acq_exposure_count,
+  DROP COLUMN c_acq_signal_to_noise;
+
+-- Add blob column for all target results in observation
+ALTER TABLE t_itc_result
+  ADD COLUMN c_asterism_results jsonb not null,
```

Comment:
Now a result has times for multiple targets. If we want individual columns, we need another table (one for the asterism request and another for the individual targets results). Since we will always read the results for the whole asterism, there doesn't seem to be much of a penalty (or any at all) for the simplicity of having everything in a json blob.